### DISCUSSION TITLE
重复充值了三年 要求退款2年

### DISCUSSION THREAD

**Discussion by lvksky** [getlantern/lantern-forum-cn#473] [General]
因为网络卡顿重复充值了三年 要求退款2年

**Comment by MaxMKMax** [comment]
@lvksky 嗯嗯. 好的. 我帮您提交错误的两年退款. 请您提供我支付单据和账号邮箱. 不好意思影响了您的购买体验. 谢谢您支持蓝灯!

**Comment by lvksky** [comment]
***@***.***


王朗聪
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月27日(星期六) 凌晨0:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 重复充值了三年 要求退款2年 (Discussion #473)





 
@lvksky 嗯嗯. 好的. 我帮您提交错误的两年退款. 请您提供我支付单据和账号邮箱. 不好意思影响了您的购买体验. 谢谢您支持蓝灯!

**Reply by MaxMKMax** [reply]
@lvksky 您好. 不好意思需要麻烦您一下. 请您发到我的邮箱给我吧. 请您提供我支付单据和账号邮箱. mike@getlantern.org
GitHub屏蔽了您的所有截图, 完全看不到. 麻烦您了. 谢谢!

![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/f0d3c2d9-3167-47a6-8aaf-20b7a43fa29d)

**Comment by lvksky** [comment]
已经回复你发的邮箱了，后续有处理了吗？



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月29日(周一) 晚上10:59
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 重复充值了三年 要求退款2年 (Discussion #473)




 
@lvksky 您好. 不好意思需要麻烦您一下. 请您发到我的邮箱给我吧. 请您提供我支付单据和账号邮箱. ***@***.***
 GitHub屏蔽了您的所有截图, 完全看不到. 麻烦您了. 谢谢!
 
image.png (view on web)

**Reply by MaxMKMax** [reply]
skywlc@qq.com
@lvksky 您好. 您的邮箱是这个吗? 已经帮您提交上去给负责任了. 请您稍等一下. 不好意思. 谢谢啦!
